### ISSUE TITLE
onboarding: Script to perform the start of the helm start

### ISSUE THREAD

**Issue by Anmol1696** [cosmology-tech/starship#73]
## Overview
Currently we are using `make` commands with helm directly to start the cluster running. With this we are not able to provide accurate information to the user with hints and suggestions

## Proposal
Create a `scripts/start.sh` which the users will provide a `config.yaml` file to spinup the helm charts. It is supposed to do the following:
1. Check access to kubernetes with `kubectl get pods`
2. Make sure all dependencies are installed (only check)
3. Warn users about the resources that are going to be consumed (nice to have feature)
4. Install helm chart
5. If resources are in pending state then we should warn the users about the resource utilization (nice to have feature)
6. Perform port-forward

This should make the onboarding much smoother.

**Comment by Anmol1696** [comment]
Not needed after #99
